File: CPP/struct1.cpp
#include <iostream>
using namespace std;

struct student {
    int id;
    char name[50];
    float marks;
};

int main() {
    struct student s1;

    cout << "Enter the id: ";
    cin >> s1.id;

    cout << "Enter the name: ";
    cin >> s1.name;

    cout << "Enter the marks: ";
    cin >> s1.marks;

    cout << "\n--- Student Details ---\n";
    cout << "ID: " << s1.id << endl;
    cout << "Name: " << s1.name << endl;
    cout << "Marks: " << s1.marks << endl;

    return 0;
}

File: Python/struct1.py
class Student:
    def __init__(self, id, name, marks):
        self.id = id
        self.name = name
        self.marks = marks

# take input
id = int(input("Enter student id: "))
name = input("Enter student name: ")
marks = float(input("Enter student marks: "))

# create object
s1 = Student(id, name, marks)

# print details
print("\n--- Student Details ---")
print("ID    :", s1.id)
print("Name  :", s1.name)
print("Marks :", s1.marks)
